content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,240 @@
+specific product. Unlike packages, packagings cannot be reusable, and each specific packaging must
+be defined on the individual product form.
+
+For example, different packages for cans of soda can be configured as a 6-pack, a 12-pack, or a case
+of 36. Each flavor of soda would need a `6`, `12`, and `36` can packaging configured on the
+individual product since packagings are product specific, not generic.
+
+Set up packagings
+-----------------
+
+To use packagings, ensure the :guilabel:`Product Packagings` feature is enabled. To do that,
+navigate to :menuselection:`Inventory app --> Configuration --> Settings`. Then, under the
+:guilabel:`Products` heading, check the box next to :guilabel:`Product Packagings`, and click
+:guilabel:`Save`.
+
+In Odoo, product packagings are used on sales/purchase orders and inventory transfers.
+
+Next, to create packagings, navigate to :menuselection:`Inventory app --> Products --> Products` and
+click on the desired product, then click :guilabel:`Edit` on the product page.
+
+Under the :guilabel:`Inventory` tab, scroll down to the :guilabel:`Packaging` section, and click
+:guilabel:`Add a line`. Then, a pop-up window appears, in which the following information for each
+packaging should be entered.
+
+- :guilabel:`Packaging`: Name of packaging that appears on sales/purchase orders as a packaging
+  option for the product.
+- :guilabel:`Contained quantity`: Amount of product in the packaging.
+- :guilabel:`Barcode`: Identifier used with the *Barcode* app to trace the packaging of a product
+  during stock moves or pickings. Leave blank if not in use.
+- :guilabel:`Company`: Indicates the packaging is only available at the selected company. Leave
+  blank to make the packaging available across all companies.
+
+.. example::
+   Create a packaging type for 6 cans of the product, `Grape Soda`, by naming the
+   :guilabel:`Packaging` to `6-pack` and setting the :guilabel:`Contained quantity` to `6` in the
+   pop-up window that appears after clicking on :guilabel:`Add a line`.
+
+   .. image:: usage/create-product-packages.png
+      :align: center
+      :alt: Create 6-pack case for product.
+
+When all the necessary information has been entered, either click :guilabel:`Save & Close` to save
+the packaging and return to the product detail form, or :guilabel:`Save & New` to save the packaging
+and create another one in a fresh pop-up window.
+
+.. example::
+   View all product packagings in the :guilabel:`Inventory` tab of the product form.
+
+   .. image:: usage/display-product-packaging.png
+      :align: center
+      :alt: Show packaging and contained quantities, specified on the product page form in the
+            Inventory tab.
+
+View all packagings
+-------------------
+
+To view all packagings that have been created, go to :menuselection:`Inventory app --> Configuration
+--> Product Packagings`. Doing so reveals a separate page with a complete list of all packagings
```

Comment:
does this "separate page" have a name?
Also, is it a page, or a _dashboard_?
it's better to be specific on things like this...